# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xen-tools/hot-fix.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur xen-3.4.2-orig/xen/crypto/vmac.c xen-3.4.2/xen/crypto/vmac.c
--- xen-3.4.2-orig/xen/crypto/vmac.c	2009-11-14 21:15:00.000000000 +0000
+++ xen-3.4.2/xen/crypto/vmac.c	2009-11-14 21:25:14.000000000 +0000
@@ -378,7 +378,7 @@
 	#define ASJ(x, y, z) __asm {x label##y}
 #endif
 
-static void NOINLINE nh_16_func(const uint64_t *mp, const uint64_t *kp, size_t nw, uint64_t *rh, uint64_t *rl)
+static void noinline nh_16_func(const uint64_t *mp, const uint64_t *kp, size_t nw, uint64_t *rh, uint64_t *rl)
 {
 	// This assembly version, using MMX registers, is just as fast as the
 	// intrinsics version (which uses XMM registers) on the Intel Core 2,
